example/mapbox.html

```diff
@@ -8,7 +8,7 @@
     <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js" integrity="sha256-hwg4gsxgFZhOsEEamdOYGBf13FyQuiTwlAQgxVSNgt4="
         crossorigin="anonymous"></script>
     <script src='../vendor/mapbox-gl-dev.js'></script>
-    <link href='https://api.tiles.mapbox.com/mapbox-gl-js/v0.42.2/mapbox-gl.css' rel='stylesheet' />
```

Comment:
Why are you copying this file on the repo?

Reply:
As said in the commit `to improve test stability and speed` we must reduce network requests.

When doing screenshot testing the browser has to download this file. 

If we have the file in the same machine, we get rid of network issues and we improve the download time (0 latency and lower download time)

Reply:
This is a 10KB sized file that is downloaded from a CDN and that probably triggers chrome's cache between the different tests. I think this is a premature optimization: the cost of downloading these is nothing compared to the MGL initialization time or the time spent with Maps API requests. As with most optimizations, this incurs in a development cost, with this we cannot try different MGL versions (which is a must) with ease.

Same argument to the other assets: they are small, they are downloaded from a CDN and they can be client-side cached by the browser.

Reply:
Any reason foto keeping them?

Reply:
> Any reason for keeping them?

eh? I don't understand the question

Reply:
Whats the reason for download those files instead having them locally?

Reply:
Development and maintenance cost: updates are more difficult to get from upstream this way and there is no good reason to add them in the first place. This optimization would be fine if it improved testing times by... let's say a 10%. But they probably improve testing times less than a 3%. If we want to improve testing times we need to profile and then optimize the more consuming things, which this I suspect is not.
Getting a renderer 'load' event would improve testing times much more and that doesn't imply any additional difficulty in testing against different MGL versions

Reply:
>  updates are more difficult to get from upstream this way

If mapbox releases an update, we should change a line for every example, keeping the file locally we only need to update this file. I don't see the extra cost

> This optimization would be fine if it improved testing times by... let's say a 10%. But they probably improve testing times less than a 3%
Is not only about time, is about stability. Network requests on a test are dangerous, IMO we should try to minimize them.

> Getting a renderer 'load' event would improve testing times much more
Agreed 100% this would be nice!

Reply:
> we should change a line for every example

That can be modified automatically by a simple replace all. Moreover, if something is failing we just need to change the example Mapbox GL CDN URL version to discard integration issues. With the assets inside the repo is much more difficult.

> is about stability. Network requests on a test are dangerous

Stability of small assets delivered over a CDN is much much higher than dynamic requests to Maps API.